egs/mgb2/ASR/conformer_ctc/conformer.py

```diff
@@ -0,0 +1,901 @@
+#!/usr/bin/env python3
+# Copyright (c)  2021  University of Chinese Academy of Sciences (author: Han Zhu)
```

Comment:
If you don't make changes to this file, could you please replace it with a symlink to the one from the librispeech recipe?

Reply:
Done